include/arch/arc/arch.h

```diff
@@ -68,8 +68,9 @@ extern "C" {
 	#define STACK_GUARD_SIZE 0
 #endif /* CONFIG_MPU_STACK_GUARD */
 
-#define STACK_SIZE_ALIGN(x)	MAX(STACK_ALIGN, x)
 
+#define STACK_SIZE_ALIGN(x) \
```

Comment:
Please use existing macros for rounding values up or down. ROUND_UP() ROUND_DOWN() in include/misc/util.h

Reply:
done